spec.md

```diff
@@ -23,42 +23,25 @@ A standard token interface is a basic building block for many applications on th
    	totalSupply : Nat; // token total supply
    	owner : Principal; // token owner
    	fee : Nat; // fee for update calls
-   	feeTo : Principal; // fee receiver
```

Comment:
I feel we are a little bit too general here :) I think this could be something that we add for ERC-20+ standard on top of the basic ERC-20.